docs/source/deployment/CORS_Headers.rst

```diff
@@ -0,0 +1,95 @@
+========================================
+Add CORS Headers to MOTECH Configuration
+========================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Add CORS support to Apache server
+=================================
```

Comment:
IMHO, this reads like its 100% generic and that its just instructions about how to configure CORs headers...

Perhaps what really needs to happen is we need to assert how to install MOTECH on Tomcat/Jetty and where to set up the MOTECH-UI. I feel like we have avoided that question in previous discussions... and this is probably a different ticket/documentation item

Question: When would we run MOTECH behind Apache? Could we just focus on Tomcat/Jetty? I'm not 100% sure about this, but shouldn't headers set in Apache/Jetty 'bubble up' through Apache even if we set it up that way?